_posts/2016/05/2016-05-31-what-digital-humanists-also-do.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+authors: ["Kathy Chung"]
+title: "What Digital Humanists Also Do"
+date: 2016-05-30
+time: "00:05:00"
+category: ["Digital Humanities"]
+---
+Following up on [an earlier post]({{site.baseurl}}/blog/2016/05/what-digital-humanists-do.html),
+here are two more user stories from digital humanists
```

Comment:
Sorry about the confusion.  It's actually three different scholars' stories.  Gamma told me two stories, one about a senior scholar (not Gamma) working on architecture and one about themselves (Gamma) working on metal work containers.